linera-core/src/client/mod.rs

```diff
@@ -603,6 +599,45 @@ pub enum ChainClientError {
     },
 }
 
+impl From<NodeError> for ChainClientError {
+    fn from(error: NodeError) -> Self {
+        match error {
+            NodeError::BlobsNotFound(blob_ids) => Self::BlobsNotFound(blob_ids),
+            error => Self::RemoteNodeError(error),
+        }
+    }
+}
```

Comment:
Actually, I think this is a conversion we should _not_ make, since we will want to distinguish: If the error is that the _remote_ node doesn't have a blob, we need to send it. If the error is that _we_ don't have a blob, we need to request it or fail. (See https://github.com/linera-io/linera-protocol/pull/2871.)